### DISCUSSION TITLE
Welcome to the UCAN Invocation Discussions!

### DISCUSSION THREAD

**Discussion by expede** [ucan-wg/invocation#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by expede** [comment]
@simonwo let's get this party started 🎉

**Comment by simonwo** [comment]
Hello! I'm @simonwo working with @filecoin-project on [Bacalhau](https://github.com/filecoin-project/bacalhau).

I'm contributing to this spec with the hope that we can:
- Have a cross-network results caching layer, on which we can build memoization, data incentivisation and reputation heuristics
- Have a way to call jobs on other networks (Bacalhau especially ofc!) from within IPVM

**Comment by andrewzhurov** [comment]
Heyo, folks! Name's Andrew. Background in web dev with Clojure. Presently I'm seeking a more robust foundation for humanity's knowledge, the idea of having reified compute, preserving computation's as the source of truth in content-addressable manner, opposite to data derived from it (state of a db), fascinates me. It's been a pleasure to find a community of like-minded people that put a ton of effort in pioneering next-gen solutions such as UCAN, WNFS, Rhizome, IPVM & homestar - gosh, there's a ton of sick tech! I hope to learn a thing or two from you, great folks, and perhaps contribute something of value along the way. :smile:
